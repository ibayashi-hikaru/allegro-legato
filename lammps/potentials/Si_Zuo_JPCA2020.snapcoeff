# DATE: 2020-01-31 UNITS: metal CONTRIBUTOR: Yunxing Zuo <y9zuo@eng.ucsd.edu> CITATION: Y. Zuo, C. Chen, X, Li, Z. Deng, Y. Chen, J. Behler, G. Cs√°nyi, A. V. Shapeev, A. P. Thompson, M. A. Wood, S. P. Ong, "Performance and Cost Assessment of Machine Learning Interatomic Potentials" J. Phys. Chem. A 124, 4, 731-745 (2020)
# Generated by Materials Virtual Lab

1 56
Si 0.5 1
-5.663990890722773
2.135204505942156e-05
0.01812688733288438
0.05890084411385875
-0.016040522397783287
0.2494122937439849
0.030685603182186407
0.07897074613855619
0.03775626980109443
0.08012875460906028
0.11447594456727353
0.08770261320757494
-0.014534999402703207
-0.016756593957058016
0.011119293334051157
0.015633833128521908
0.026807206079160067
0.04870622354746678
-0.06492810262901873
0.002910199026683352
-0.019908133833638274
-0.017872613445740447
0.0031311465759044123
0.011978920186307855
0.0036899231853172246
0.03006578350372085
0.021354421560243725
-0.03643908247119505
-0.00831639295486291
-0.04990225691971903
-0.0010633320986013165
-0.03193409548541607
-0.003055748773152782
0.002573793612554956
0.0044391007289529515
0.0325144360034856
0.005840334419032887
-0.004912536969652485
-0.016370550877355387
0.002140265199864603
-0.027229855984721617
-0.008278354893853326
-0.006191386043061048
0.0006411177578441998
0.00552251715009933
0.021112108349453373
-0.0072983636775871275
-0.002894532519792062
-0.011272781710144501
-0.008790720107990663
-0.0069330757563178524
0.0012142867357010229
0.007119555811982486
0.0010458619432982784
0.006896454998006431
-0.0034341539192112164
