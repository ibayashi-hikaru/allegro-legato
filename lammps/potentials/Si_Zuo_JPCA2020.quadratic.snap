# DATE: 2020-01-31 UNITS: metal CONTRIBUTOR: Yunxing Zuo <y9zuo@eng.ucsd.edu> CITATION: Y. Zuo, C. Chen, X, Li, Z. Deng, Y. Chen, J. Behler, G. Cs√°nyi, A. V. Shapeev, A. P. Thompson, M. A. Wood, S. P. Ong, "Performance and Cost Assessment of Machine Learning Interatomic Potentials" J. Phys. Chem. A 124, 4, 731-745 (2020)
# Generated by Materials Virtual Lab

pair_style snap
pair_coeff * * Si_Zuo_JPCA2020.quadratic.snapcoeff Si_Zuo_JPCA2020.quadratic.snapparam Si
